src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <Modal
+    full-screen
+    from-bottom
+    no-padding
```

Comment:
I strongly suggest to avoid removing the modal padding and set it again with additional CSS.